test/include/seqan3/test/performance/naive_minimiser_hash.hpp

```diff
@@ -65,13 +67,27 @@ struct naive_minimiser_hash_fn
        if (shape.size() > window_size)
            throw std::invalid_argument{"The size of the shape cannot be greater than the window size."};
 
-       return std::forward<urng_t>(urange) | seqan3::views::kmer_hash(shape)
-                                           | std::views::transform([seed] (uint64_t i) { return i ^ seed; })
-                                           | ranges::view::sliding(window_size - shape.size() + 1)
-                                           | std::views::transform([] (auto const in)
-                                                                   {
-                                                                       return *std::min_element(in.begin(), in.end());
-                                                                   });
+       // Use random seed to randomise order on forward strand
+       auto forward = std::forward<urng_t>(urange) | seqan3::views::kmer_hash(shape)
+                                                   | std::views::transform([seed] (uint64_t const i)
+                                                                           { return i ^ seed; });
+       // Create reverse complement strand and use random seed to randomise order on reverse complement strand
+       auto reverse = std::forward<urng_t>(urange) | seqan3::views::complement // Create complement
+                                                   | std::views::reverse       // Reverse order
+                                                   | seqan3::views::kmer_hash(shape) // Get hash values
+                                                   | std::views::transform([seed] (uint64_t const i)
+                                                                           { return i ^ seed; }) // Randomise
+                                                   | std::views::reverse; // Reverse again, so that the first hash value
+                                                                          // is the reverse complement of the first
+                                                                          // hash value in the forward strand.
```

Comment:
```suggestion
       auto reverse = std::forward<urng_t>(urange) | seqan3::views::complement // Create complement.
                                                   | std::views::reverse       // Reverse order.
                                                   | seqan3::views::kmer_hash(shape) // Get hash values.
                                                   | std::views::transform([seed] (uint64_t const i)
                                                                           { return i ^ seed; }) // Randomise.
                                                   | std::views::reverse; // Reverse again, so that the first hash value.
                                                                          // is the reverse complement of the first.
                                                                          // hash value in the forward strand.
```

Reply:
// Reverse again, so that the first hash value.
 // is the reverse complement of the first.

The dots here make no sense because the sentence ends in the line below.

Reply:
right sorry.